lspserver/include/lspserver/LSPServer.h

```diff
@@ -66,7 +66,8 @@ class LSPServer : public MessageHandler {
 
   template <class ParamTy, class ResponseTy>
   llvm::unique_function<void(const ParamTy &, Callback<ResponseTy>)>
-  mkOutMethod(llvm::StringRef Method, OutboundPort *O = nullptr) {
+  mkOutMethod(llvm::StringRef Method, OutboundPort *O = nullptr,
+              ResponseTy Base = ResponseTy()) {
```

Comment:
same as above

Reply:
I agree that this doesn't make much sense so I will try to move the behaviour to Controller.cpp.